Packages/Models/Sources/Models/Card.swift

```diff
@@ -10,6 +10,8 @@ public struct Card: Codable, Identifiable, Equatable, Hashable {
   public let description: String?
   public let type: String
   public let image: URL?
+  public let width: CGFloat
+  public let height: CGFloat
```

Comment:
Are you confident it's not optional? Maybe it should so we don't crash the whole feed.

Reply:
yes, I've read the API doc

![image](https://github.com/Dimillian/IceCubesApp/assets/46838577/59747d2e-81ce-45fe-80f7-ca40f9ab1094)